lxd/images.go

```diff
@@ -3019,16 +3019,16 @@ func imageGet(d *Daemon, r *http.Request) response.Response {
 		userCanViewImage = true
 	}
 
-	public := d.checkTrustedClient(r) != nil || !userCanViewImage
```

Comment:
Where is the equivalent of d.checkTrustedClient happening now?

It would be helpful to explain more in the commit message.

Reply:
In this case `userCanViewImage=false` means that the caller does not have permission, regardless of whether they are authenticated or not. `userCanViewImage=true` means that the caller is authenticated and does have permission. 

So in this case `public` is now equivalent to `!userCanViewImage`. Shall I follow up with more comments?

Reply:
If you can that would be helpful i think for future readers. Thanks